.github/workflows/release.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         working-directory: ./
         run: |
           echo "Checking all contracts under ./artifacts"
-          docker run --volumes-from with_code rust:1.57.0 /bin/bash -e -c 'cd ./code/packages/vm; ./examples/check_contract.sh ../../artifacts/*.wasm'
+          docker run --volumes-from with_code rust:1.57.0 /bin/bash -e -c 'cd ./code/packages/vm; export GLOBIGNORE=../../artifacts/floaty.wasm; ./examples/check_contract.sh ../../artifacts/*.wasm; export GLOBIGNORE='
```

Comment:
we need this line? `; export GLOBIGNORE=`
It seems just in the docker env. So I guess I can just leave it alone.

Reply:
Correct. thank you.